src/main/resources/reference.conf

```diff
@@ -24,6 +24,20 @@ akka {
         # How often to sweep out expired cache entries.
         # Note that this interval has nothing to do with TTLs
         cache-cleanup-interval = 120s
+
+        # The IPs of nameservers to use e.g.
+        # akka.io.dns.async-dns {
+        #   nameservers = ["8.8.8.8", "8.8.4.4"]
+        # }
+        nameservers = []
+
+        # Whether to use the system's /etc/resolv.conf in place
+        # of nameservers
+        resolv-conf = off
+
+        # The time that a request is allowed to live before being discarded
+        # given no reply
+        request-ttl = 5s
```

Comment:
Could you have a go at re-wording my comment (which is doc)? ...or are you referring to some other doc?